### ISSUE TITLE
$SSH_AUTH_SOCK and $SSH_AGENT_PID are not set

### ISSUE THREAD

**Issue by theRealBithive** [dyuri/xontrib-ssh-agent#5]
$SSH_AUTH_SOCK and $SSH_AGENT_PID are not set on my system.
I use ArchLinux and AwesomeWM as Window Manager - so ssh-agent is not started on startup and the variables are not set on startup.

So I get: 

```
$ ssh-add -L
Could not open a connection to your authentication agent.
```

**Comment by dyuri** [comment]
Indeed they aren't currently, I'll try to add them.

**Comment by dyuri** [comment]
`1.0.13` released, requires `repassh>=1.2.0`, please check if you have time.

**Comment by theRealBithive** [comment]
It's working! Thanks for implementing it!
